infra/base-images/base-runner/python_coverage_runner_help.py

```diff
@@ -59,6 +60,37 @@ def translate_coverage(all_file_paths):
   covdata_post_translation.write()
 
 
+def convert_coveragepy_cov_to_summary_json(src, dst):
+  """
+  Convert a json file produced by coveragepy into a summary.json file
+  similary to llvm-cov output. `src` is the source coveragepy json file,
+  `dst` is the destination json file, which will be overwritten.
+  """
+  dst_dict = {'data': {'files': dict()}}
+  with open(src, "r") as src_f:
+    src_json = json.loads(src_f.read())
+    for elem in src_json['files']:
+      src_dict = src_json['files'][elem]['summary']
```

Comment:
Do we need more checks to guard against potentially malformed inputs, or incorrect assumptions about fields existing?

Reply:
The idea was that a hard exit is preferred as I assume errors would be due to solvable failures in a given project's coverage build, and we'd like to know when those happen so they can be fixed.

That said, it might have been a bit too assuming: I added guards now for checking the top elements are in order in the src json file now.